packages/terra-site/src/examples/card/Index.jsx

```diff
@@ -24,16 +24,18 @@ const CardExamples = () => (
     <Markdown id="readme" src={ReadMe} />
     <PropsTable componentName="Card" id="props" src={CardSrc} />
     <PropsTable componentName="Card Body"src={CardBodySrc} />
-    <h2 id="CardDefault">Card basic example</h2>
-    <CardDefault />
-    <h2 id="CardPadding">Card plus Card Body with default padding</h2>
-    <CardPadding />
-    <h2 id="CardPaddingVertical">Card plus Card Body with only vertical padding</h2>
-    <CardPaddingVertical />
-    <h2 id="CardPaddingHorizontal">Card plus Card Body with only horizontal padding</h2>
-    <CardPaddingHorizontal />
-    <h2 id="CardPaddingHR">Card plus padded and non-padded children</h2>
-    <CardPaddingHR />
+    <ThemeProvider>
```

Comment:
Remove theme provider, it is not needed here. The theme provider is provided at a higher level in terra-site/src/App.jsx I believe.